formulaire/src/main/java/fr/openent/formulaire/service/impl/DefaultFolderService.java

```diff
@@ -39,6 +43,16 @@ public void get(String folderId, Handler<Either<String, JsonObject>> handler) {
         Sql.getInstance().prepared(query, params, SqlResult.validUniqueResultHandler(handler));
     }
 
+    @Override
+    public Future<JsonObject> get(String folderId, UserInfos user) {
+        Promise<JsonObject> promise = Promise.promise();
+
+        String errorMessage = "[Formulaire@DefaultFolderService::get] Fail to get folder with id " + folderId + " : ";
+        get(folderId, FutureHelper.handlerEither(promise, errorMessage));
+
+        return promise.future();
+    }
+
```

Comment:
Pour guidé les gens a utilisé la version des futures

```suggestion
    /**
     * @deprecated Use {@link #get(String)}
     */
    @Override
    @Deprecated
    public void get(String folderId, Handler<Either<String, JsonObject>> handler) {
           this.get(folderId)
               .onSuccess(res -> handler.handle(new Either.Right(res)))
               .onFailure(error -> handler.handle(new Either.Left(error.getMessage())))
    }

    @Override
    public Future<JsonObject> get(String folderId, UserInfos user) {
        Promise<JsonObject> promise = Promise.promise();
String errorMessage = "[Formulaire@DefaultFolderService::get] Fail to get folder with id " + folderId + " : ";
        String query = "SELECT * FROM " + FOLDER_TABLE + " WHERE id = ?;";
        JsonArray params = new JsonArray().add(folderId);
        Sql.getInstance().prepared(query, params, SqlResult.validUniqueResultHandler(FutureHelper.handlerEither(promise, errorMessage)));

        return promise.future();
    }

```

De plus un model de la table FOLDER_TABLE serait pas mal.
Tu as réellement besoin de l'user en paramètre de fonction ?